fili-system-config/pom.xml

```diff
@@ -62,15 +68,35 @@
                 </executions>
             </plugin>
             <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
+                <version>1.12</version>
+                <executions>
+                    <execution>
+                        <id>moduleJars</id>
+                        <goals>
+                            <goal>add-test-resource</goal>
+                        </goals>
+                        <configuration>
+                            <resources>
+                                <resource>
+                                    <directory>target/moduleJars</directory>
+                                </resource>
+                            </resources>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
                     <additionalClasspathElements>
                         <!-- One set of paths makes maven happy, the other set makes the IDE happy -->
-                        <additionalClasspathElement>${basedir}/src/test/resources/jars/fili-system-config-test1.jar</additionalClasspathElement>
-                        <additionalClasspathElement>${basedir}/src/test/resources/jars/fili-system-config-test2.jar</additionalClasspathElement>
-                        <additionalClasspathElement>src/test/resources/jars/fili-system-config-test1.jar</additionalClasspathElement>
-                        <additionalClasspathElement>src/test/resources/jars/fili-system-config-test2.jar</additionalClasspathElement>
+                        <additionalClasspathElement>${basedir}/target/moduleJars/jars/fili-system-config-test1.jar</additionalClasspathElement>
+                        <additionalClasspathElement>${basedir}/target/moduleJars/jars/fili-system-config-test2.jar</additionalClasspathElement>
```

Comment:
We should be consistent to use `${project.build.directory}` here, it seems like `${basedir}` is deprecated now.